src/liblsquic/lsquic_cubic.c

```diff
@@ -133,10 +136,10 @@ lsquic_cubic_was_quiet (struct lsquic_cubic *cubic, lsquic_time_t now)
 
 
 void
-lsquic_cubic_ack (struct lsquic_cubic *cubic, lsquic_time_t now,
+lsquic_cubic_ack (struct lsquic_cubic *cubic, lsquic_time_t now_time,
```

Comment:
This change and the similar one below it is to remove a warning from the Windows compile in LOG_CWND.  There you have a scoped variable with the name 'now'.  Ought to be fine because it's scoped.  But it's Windows.  I recommend this change remain.